<launch>
	<node name="newik" pkg="newik" type="trajectory_ik_server" respawn="false" output="screen"/>
</launch>
